ddtrace/contrib/django/utils.py

```diff
@@ -303,6 +306,25 @@ def _after_request_tags(pin, span, request, response):
             raw_uri = url
             if raw_uri and request.META.get("QUERY_STRING"):
                 raw_uri += "?" + request.META["QUERY_STRING"]
+
+            req_body = None
+
+            if _appsec_enabled and request.method in _BODY_METHODS:
+                content_type = request.content_type
+                rest_framework = hasattr(request, "data")
+
+                if content_type == "application/x-www-form-urlencoded":
+                    req_body = request.data.dict() if rest_framework else request.POST.dict()
+                elif content_type == "application/json":
+                    req_body = (
+                        json.loads(request.body.decode("UTF-8"))
+                        if rest_framework
+                        else json.loads(request.body.decode("UTF-8"))
```

Comment:
I don't see the need for the if else here, both do `json.loads(request.body.decode("UTF-8"))` right?

Reply:
Maybe it's a typo, and you meant `request.data.decode...` for the first one?

Reply:
Woops, good catch!

Reply:
Fixed.